### PR TITLE
run tests in travis

### PR DESCRIPTION
**Pull request by aweyl**
1. make changes in travis.yml and makefile
2. change the flow tests to run with disposable redis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,13 +15,8 @@ before_script:
...
 
 script:
-  - make clean
-  - make
-  - cd src
-  - mkdir -p ../build
-  - module_packer -v -o "../build/redisgraph.{os}-{architecture}.latest.zip" "`pwd`/redisgraph.so"
-  - cd ..
-  # - make test
+  - make test
```
Comment (swilly22) [inline .travis.yml]: What about simply calling `make` to build the module?

Comment (swilly22) [reply .travis.yml]: Also I don't think test target is defined within our makefile(s)

Comment (aweyl) [reply .travis.yml]: 1. Right, I forgot to define it.
2. I just made it the same way it was done in redisearch. They call make in before_script, and then make test and make package in script

Comment (swilly22) [reply .travis.yml]: make in the before script builds redis,
I don't think our module is getting built.

File: tests/flow/base.py [hunk 2]
```diff
@@ -1,13 +1,7 @@
...
+from rmtest import ModuleTestCase
 
 
-class FlowTestsBase(unittest.TestCase):
-    RUN_TIME_ROW_INDEX = 4
-
-    @classmethod
-    def setUpClass(cls):
-        cls.redis_con = redis.Redis(host='localhost', port=6379)
+class FlowTestsBase(ModuleTestCase('../../src/libmodule.so')):
```
Comment (swilly22) [inline tests/flow/base.py]: Change to redisgraph.so

File: src/Makefile [hunk 3]
```diff
@@ -89,4 +89,8 @@ redisgraph.so: $(MODULE)
 	$(LD) -o $@ $(MODULE) $(SHOBJ_LDFLAGS) -lc -lm
 
 clean:
-	find . -name '*.[oad]' -type f -delete
\ No newline at end of file
+	find . -name '*.[oad]' -type f -delete
+
+package: redisearch.so
```
Comment (swilly22) [inline src/Makefile]: Depends on redisgraph.so

**Pull request merged**